src/Node.jsx

```diff
@@ -42,7 +42,7 @@ class Node extends React.Component {
   }
 
   famousDelete() {
-    this._famousNode.dismount();
+    this._famousNode.getParent().removeChild(this._famousNode);
```

Comment:
@pilwon By the way, I've ben using react-famous on two projects. It's really nice (except for all the bugs in Famous itself right now)!